spec/jobs/create_case_comment_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../spec_helper'
+
+RSpec.describe Jobs::CreateCaseComment do
+  include_context "spec helper"
+
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:_case) { Fabricate(:salesforce_case, topic: topic) }
+  fab!(:post) { Fabricate(:post, topic: topic) }
+
+  it 'creates a case comment object on Salesforce' do
```

Comment:
There are multiple conditions in the job which does not create a case comment. I think we should test for those too.